source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/admin/index.tsx

```diff
@@ -4,17 +4,16 @@ import { Switch, Route } from 'react-router-dom';
 import { requireAuth } from '@lib/auth';
 import { withRole } from '@data/containers/with-role';
 import { ROLE } from '@data/models/role';
+import { adminPath } from '@ui/routes/paths';
 
 import InviteOrganization from './invite-organization';
 import AdminSettingsRoute, { pathName as settingsPathName } from './settings';
 
-export const pathName = '/admin';
-
 class AdminRoute extends React.Component {
   render() {
     return (
       <Switch>
-        <Route exact path={pathName} component={InviteOrganization} />
+        <Route exact path={adminPath} component={InviteOrganization} />
```

Comment:
all path changes were due to experimenting with lazy-loading / code-splitting on routes early on in this PR when I was blocked from actually working on the language picker.

I learned some things about code splitting, which I don't think is well known: https://twitter.com/nullvoxpopuli/status/1093512581896114176
basically, code-splitting isn't always beneficial. testing to see if it has any benefit is a good thing to do.